### PR TITLE
Adding st2scheduler

### PR DESCRIPTION
**Pull request by bigmstone**
This PR adds the initial break out of the scheduler. This PR will only encompass breaking out the service. Further enhancements will happen in subsequent PRs.

### CODE REVIEW

File: conf/logrotate.conf [hunk 1]
```diff
@@ -110,6 +110,15 @@ notifempty
   endscript
 }
 
+## Scheduler
+/var/log/st2/scheduler*.log {
```
Comment (armab) [inline conf/logrotate.conf]: `/var/log/st2scheduler` instead of `/var/log/scheduler`

File: conf/st2.package.conf [hunk 2]
```diff
@@ -83,3 +83,7 @@ remote_dir = /tmp
 
 [workflow_engine]
 logging = /etc/st2/logging.workflowengine.conf
+
+[scheduler]
+logging = /etc/st2/logging.st2scheduler.conf
```
Comment (armab) [inline conf/st2.package.conf]: `logging.scheduler.conf` instead of `logging.st2scheduler.conf`

File: conf/st2.prod.conf [hunk 3]
```diff
@@ -79,3 +79,6 @@ logging = /etc/st2timersengine/logging.st2timersengine.conf
 
 [workflow_engine]
 logging = /etc/st2actions/logging.workflowengine.conf
+
+[scheduler]
+logging = /etc/st2actions/logging.st2scheduler.conf
```
Comment (armab) [inline conf/st2.prod.conf]: Same typo

**Comment by Kami** [comment]
We also need to update documentation for how - describe the new component, update service diagram, etc.

**Pull request merged**